dotcom-rendering/src/components/marketing/epics/ContributionsLiveblogEpic.tsx

```diff
@@ -60,17 +62,9 @@ const container = (tracking: Tracking, isInTestVariant: boolean) => css`
 		}
 	}
 
-	& > * + * {
-		margin-top: ${space[3]}px;
```

Comment:
these child margins were not actually having any effect because each existing component defined its own margin-top, overriding this.